code/modules/client/preference/preferences_spawnpoints.dm

```diff
@@ -28,30 +28,4 @@ var/list/spawntypes = list()
 
 /datum/spawnpoint/arrivals/New()
 	..()
-	turfs = latejoin
-
-/datum/spawnpoint/gateway
-	display_name = "Gateway"
-	msg = "has completed translation from offsite gateway"
-
-/datum/spawnpoint/gateway/New()
-	..()
-	turfs = latejoin_gateway
-
-/datum/spawnpoint/cryo
-	display_name = "Cryogenic Storage"
-	msg = "has completed cryogenic revival"
-	disallow_job = list("Cyborg")
-
-/datum/spawnpoint/cryo/New()
-	..()
-	turfs = latejoin_cryo
-
-/datum/spawnpoint/cyborg
-	display_name = "Cyborg Storage"
-	msg = "has been activated from storage"
-	restrict_job = list("Cyborg")
-
-/datum/spawnpoint/cyborg/New()
```

Comment:
Functionality doesn't exist anymore. Removing dead code.

Reply:
This functionality absolutely exists, do not remove this code. Literally everything is present.... but, for some reason (I believe at the time we couldn't decide on where to put alternative spawns), the code for selecting your spawnpoint is commented out in preferences.dm.

Reply:
Looking further, I'm pretty sure MarkvA just didn't want to implement the sql saving at the time and then... never got to it.

Reply:
@tigercat2000 Right. So it doesn't exist.